specification/redis/resource-manager/Microsoft.Cache/preview/2023-05-01/redis.json

```diff
@@ -2697,6 +3007,167 @@
+        },
+        "permissions": {
+          "type": "string",
+          "description": "Command permissions for the access policy."
+        }
+      },
+      "required": [
+        "permissions"
+      ],
+      "description": "All properties of an access policy.",
+      "type": "object"
+    },
+    "RedisCacheAccessPolicyRequestResponse": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/RedisCacheAccessPolicyProperties",
+          "description": "Properties of an access policy."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "description": "Response to get/put access policy.",
+      "type": "object"
+    },
+    "RedisCacheAccessPolicyListResponse": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RedisCacheAccessPolicyRequestResponse"
+          },
+          "description": "List of access policies (with properties) of a Redis cache."
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Link for next set."
+        }
+      },
+      "description": "List of access policies (with properties) of a Redis cache.",
+      "type": "object"
+    },
+    "RedisCacheAccessPolicyAssignment": {
+      "description": "Properties for an access policy assignment",
+      "properties": {
+        "objectId": {
+          "type": "string",
+          "description": "Object Id to assign access policy to"
+        },
+        "objectIdAlias": {
+          "type": "string",
+          "description": "User friendly name for object id. Also represents username for token based authentication"
+        }
+      },
+      "required": [
+        "objectId",
+        "objectIdAlias"
+      ],
+      "type": "object"
+    },
+    "RedisCacheAccessPolicyAssignmentProperties": {
+      "properties": {
+        "provisioningState": {
+          "type": "string",
+          "description": "Provisioning state of an access policy assignment",
+          "readOnly": true,
+          "enum": [
+            "Updating",
+            "Succeeded",
+            "Deleting",
+            "Deleted",
+            "Canceled",
+            "Failed"
+          ],
+          "x-ms-enum": {
+            "name": "AccessPolicyAssignmentProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "assignments": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RedisCacheAccessPolicyAssignment"
+          },
+          "x-ms-identifiers": [
+            "objectId"
+          ]
+        }
+      },
+      "type": "object",
+      "description": "Properties of an access policy assignment"
+    },
+    "RedisCacheAccessPolicyAssignmentRequestResponse": {
```

Comment:
>RedisCacheAccessPolicyAssignmentRequestResponse [](http://example.com/codeflow?start=5&length=47)

So how is this different from an assignment... sems like the main actual difference is that... the properties actually contains a LIST of assignments. 
So should we call this instead a redisCacheAccessPolicyAssignmentSet??

Reply:
Renamed to RedisCacheAccessPolicyAssignmentSet